.github/workflows/ci-cd.yml

```diff
@@ -62,6 +62,7 @@ jobs:
           cd $GITHUB_WORKSPACE
           if [[ $OS == "linux" && $ARCH == "amd64" ]]; then
             make OS=$OS ARCH=$ARCH
+            sudo env "PATH=$PATH" make privileged-test covhtml
```

Comment:
No need to call 'covhtml' target here, the coverage ci-cd will pick up an *.txt as coverage.

Reply:
And in any case, add this file to `covhtml` target in the Makefile just to be consistent.

Reply:
> No need to call 'covhtml' target here, the coverage ci-cd will pick up an *.txt as coverage.

Removed covhtml from that line. I had put it there as I didn't know whether the coverage ci-cd would pick up the new needprivilege cover profile too ( -coverprofile=coverage-dev-needprivileges.txt in the new make target) before the covhtml target was run. The modification mentioned below makes covhtml include that file too. Or perhaps I should instead put one of the existing cover profiles when running the privileged-test target, and not a new one?

> And in any case, add this file to `covhtml` target in the Makefile just to be consistent.

Please check line 87 in the Makefile. Is that option agreeable?